suite-native/config/src/environment.ts

```diff
@@ -1,7 +1,20 @@
-export const isDebugEnv = () => process.env.EXPO_PUBLIC_ENVIRONMENT === 'debug';
-export const isDevelopEnv = () => process.env.EXPO_PUBLIC_ENVIRONMENT === 'develop';
-export const isProduction = () => process.env.EXPO_PUBLIC_ENVIRONMENT === 'production';
+import { z } from 'zod';
+
+const envVariables = z.object({
+    EXPO_PUBLIC_ENVIRONMENT: z.string(),
+    EXPO_PUBLIC_JWS_PUBLIC_KEY: z.string(),
+    EXPO_PUBLIC_CODESIGN_BUILD: z.string(),
+});
+
+// This will fail in runtime if process.env doesn't satisfy validation
+const env = envVariables.parse(process.env);
```

Comment:
This must not fail if there are some other variables on process.env (like NODE_ENVIRONMENT). I don't know if it would fall this way, just pointing it out.

Reply:
Tested locally and it doesn't fail.

Reply:
I think it would fail only if the variables defined in `envVariables` wouldn't be present.